src/_posts/2020-05-19-promises-and-lies.md

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
Please ensure that external links open in a new tab.

Reply:
> Please ensure that external links open in a new tab. 

This happens automatically now on all external links of the website, assuming the browser has not disabled JavaScript